src/TConsole.cpp

```diff
@@ -2241,7 +2241,7 @@ TConsole* TConsole::createMiniConsole(const QString& windowname,const QString& n
         }else{
```

Comment:
Uuugg, can we have a little more space!
Please change:
```cpp
        }else{
```
to:
```cpp
        } else {
```

Reply:
Yeah I'm surprised clang-format didn't pick this up, or it wasn't run on this particular line.